src/LtiGrade.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Packback\Lti1p3;
 
+use DateTime;
+use InvalidArgumentException;
+
 class LtiGrade
```

Comment:
I'm wondering whether it would make sense to add something along the lines of `TIMESTAMP_FORMAT` to this class or elsewhere. In our application, I've added a `DateTime`-based `const TIMESTAMP_FORMAT = 'Y-m-d\TH:i:s.vP'` constant.